### ISSUE TITLE
`colorcolumn` adds trailing whitespace to any multiline copy-pasted content

### ISSUE THREAD

**Issue by nschloe** [neovim/neovim#1995]
To reproduce:
1. Add `set cc=80` to your `~/.nvimrc`
2. Open any document (e.g., `~/.nvimrc`)
3. Copy-and-paste code from there (e.g., into the same file)

You'll find that each copy-and-pasted line has trailing whitespace, namely the space from the last character in a line to the color column.

See https://code.google.com/p/vim/issues/detail?id=332&thanks=332&ts=1424100587.

**Comment by lahwran** [comment]
I can't reproduce this. I tried yanking and pasting, and it behaved correctly.

Colorcolumn does look kinda funky, though: http://i.imgur.com/IQQPk7a.png

**Comment by lqueryvg** [comment]
I've been experiencing a similar problem. When I tried to chop down my vim config I found it was something to do with the nanotech/jellybeans colour scheme. There's a line which says...

`call s:X("Normal","e8e8d3",g:jellybeans_background_color,"","White","")`

which seems to cause the trouble. I'm afraid I've not had time to pin it down further than that, and I have to switch back to vim now. NVIM 0.1.2. CentOS Linux release 7.2.1511 (Core).
